NewForce-at-Mountwest/bangazon-site-pig-pong
# Lm add a product

Description
Sell A Product for a user
Related Ticket(s)
#1
Steps to Test

- If Database is not up to date then open Package Manager Console and type Update-Database and press enter
- Press green play button at the top of VS

- in the provided webpage register and login if you haven't already

- Click on the Sell A Product button in the navbar

- Fill in the required fields and Press Sell button

- You should be redirected to the Details Page of that Product
